### DISCUSSION TITLE
Audio distortion when "Use fixed size outbound buffers"

### DISCUSSION THREAD

**Discussion by jaacMusic** [apohl79/audiogridder#1052] [Help] [answered]
Hello,
I just started testing beta 14 (Windows 11) and I just realize that when "Use fixed size outbound buffers" option is enabled under Buffers it would cause audio distortion. I assume this is related to my sound interface (Focusrite) not allowing some buffer sizes.
Has anyone experienced this issue and have any comments on what this really does?

Thanks!
Juan

**Comment by apohl79** [comment, marked as answer]
No it is a bug that I fixed a few days back. Not your interface. Cheers!
